### PR TITLE
Add an option to perform connect operations in blocking mode

### PR DESCRIPTION
**Pull request by nezihyigitbasi**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -5,6 +5,7 @@
   resolution to the executor for IP address literals.
 - Rename HTTP client selector config parameter to "http-client.selector-count".
 - Add an option to disable recording HTTP request complete information.
+- Add an option to perform connect operations in blocking mode.
```
Comment (electrum) [inline CHANGES]: Add "HTTP client"

File: http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java [hunk 2]
```diff
@@ -61,6 +61,7 @@
     private String kerberosRemoteServiceName;
     private int selectorCount = 2;
     private boolean recordRequestComplete = true;
+    private boolean connectOperationsBlocking;
```
Comment (electrum) [inline http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java]: How about `blockingConnect`?

**Comment by nezihyigitbasi** [comment]
Comments addressed.

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**